cmake/configs/base.cmake

```diff
@@ -140,6 +140,16 @@ config_bool(
     DEPENDS_ERROR ON "Failed to find lz4 library"
 )
 
+config_bool(
+    ENABLE_MEMKIND
+    "Use MEMKIND as the backend allocator"
```

Comment:
@fedorova, what would be a better description?

Reply:
A better description would be: "Enable the memkind library, needed for NVRAM or SSD block caches."

Reply:
Thanks!